### ISSUE TITLE
Is first-touch build broken?

### ISSUE THREAD

**Issue by udzura** [haconiwa/haconiwa#82]
On Ubuntu Trusty(CI), `rake compile` will fail (via both gcc/clang).

https://travis-ci.org/haconiwa/haconiwa/jobs/202496047

```console
/bin/bash ../libtool --tag=CC   --mode=link gcc -g -std=gnu99 -O3 -Wall -Werror-implicit-function-declaration -Wdeclaration-after-statement -Wwrite-strings  -g -O2   -o crnl crnl.o ../libonig.la -lpthread 

libtool: link: gcc -g -std=gnu99 -O3 -Wall -Werror-implicit-function-declaration -Wdeclaration-after-statement -Wwrite-strings -g -O2 -o crnl crnl.o  ../.libs/libonig.a -lpthread

make[2]: Leaving directory `/home/travis/build/haconiwa/haconiwa/mruby/build/x86_64-pc-linux-gnu_mirb/mrbgems/mruby-onig-regexp/Onigmo-Onigmo-5.15.0/sample'

make[1]: Leaving directory `/home/travis/build/haconiwa/haconiwa/mruby/build/x86_64-pc-linux-gnu_mirb/mrbgems/mruby-onig-regexp/Onigmo-Onigmo-5.15.0'

CC    build/mrbgems/mruby-onig-regexp/src/mruby_onig_regexp.c -> build/host/mrbgems/mruby-onig-regexp/src/mruby_onig_regexp.o

/home/travis/build/haconiwa/haconiwa/mruby/build/mrbgems/mruby-onig-regexp/src/mruby_onig_regexp.c:43:23: fatal error: oniguruma.h: No such file or directory

 #include "oniguruma.h"

                       ^

compilation terminated.

/home/travis/build/haconiwa/haconiwa/mruby/build/mrbgems/mruby-onig-regexp/src/mruby_onig_regexp.c:43:23: fatal error: oniguruma.h: No such file or directory

 #include "oniguruma.h"

                       ^

compilation terminated.

rake aborted!

Command failed with status (1): [gcc -g -std=gnu99 -O3 -Wall -Werror-implic...]

/home/travis/build/haconiwa/haconiwa/mruby/tasks/mruby_build_commands.rake:31:in `_run'

/home/travis/build/haconiwa/haconiwa/mruby/tasks/mruby_build_commands.rake:36:in `rescue in _run'

/home/travis/build/haconiwa/haconiwa/mruby/tasks/mruby_build_commands.rake:32:in `_run'

/home/travis/build/haconiwa/haconiwa/mruby/tasks/mruby_build_commands.rake:89:in `run'

/home/travis/build/haconiwa/haconiwa/mruby/tasks/mruby_build_commands.rake:113:in `block (2 levels) in define_rules'

Command failed with status (1): ["gcc" -g -std=gnu99 -O3 -Wall -Werror-impl...]

/home/travis/build/haconiwa/haconiwa/mruby/tasks/mruby_build_commands.rake:33:in `_run'

/home/travis/build/haconiwa/haconiwa/mruby/tasks/mruby_build_commands.rake:89:in `run'

/home/travis/build/haconiwa/haconiwa/mruby/tasks/mruby_build_commands.rake:113:in `block (2 levels) in define_rules'

Tasks: TOP => compile => /home/travis/build/haconiwa/haconiwa/mruby/bin/mruby => /home/travis/build/haconiwa/haconiwa/mruby/build/host/bin/mruby => /home/travis/build/haconiwa/haconiwa/mruby/build/host/lib/libmruby.a => /home/travis/build/haconiwa/haconiwa/mruby/build/host/mrbgems/mruby-onig-regexp/src/mruby_onig_regexp.o

(See full trace by running task with --trace)

The command "rake compile" exited with 1.
```

But passes builds on Xenial and CentOS 7. What is the difference??

**Comment by udzura** [comment]
The guess:

* Failing pont is here:
  * https://github.com/mattn/mruby-onig-regexp/blob/master/src/mruby_onig_regexp.c#L38-L44
* After https://github.com/mattn/mruby-onig-regexp/commit/662b0401f409909e106db3ba5a001a596ff07f73 onig-regexp is intended to use Onigumo 6.0.0
* But bundled Onigumo is 5.15 ?
  * https://github.com/mattn/mruby-onig-regexp/blob/master/Onigmo-5.15.0.tar.gz

**Comment by udzura** [comment]
How is changing CI platform?

* https://circleci.com/
* http://www.wercker.com/

**Comment by udzura** [comment]
fixed via #89
